quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/LoggingPropertyMappers.java

```diff
@@ -74,78 +104,27 @@ public String apply(String value, ConfigSourceInterceptorContext configSourceInt
-                        .paramLabel("default|json")
-                        .expectedValues(DEFAULT_CONSOLE_OUTPUT,"json")
-                        .transformer((value, context) -> {
-                            if(value.equals(DEFAULT_CONSOLE_OUTPUT)) {
-                                return Boolean.FALSE.toString();
-                            }
-                            return Boolean.TRUE.toString();
-                        })
-                        .build(),
-                builder().from("log-console-format")
-                        .to("quarkus.log.console.format")
-                        .defaultValue("%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n")
-                        .description("The format of unstructured console log entries. If the format has spaces in it, escape the value using \"<format>\".")
-                        .paramLabel("format")
-                        .build(),
-                builder().from("log-console-color")
-                        .to("quarkus.log.console.color")
-                        .defaultValue(Boolean.FALSE.toString())
-                        .description("Enable or disable colors when logging to console.")
-                        .paramLabel(Boolean.TRUE + "|" + Boolean.FALSE)
-                        .build(),
-                builder().from("log-console-enabled")
-                        .mapFrom("log")
-                        .to("quarkus.log.console.enable")
-                        .hidden(true)
-                        .transformer(resolveLogHandler(DEFAULT_LOG_HANDLER))
-                        .build(),
-                builder().from("log-file-enabled")
-                        .mapFrom("log")
-                        .to("quarkus.log.file.enable")
-                        .hidden(true)
-                        .transformer(resolveLogHandler("file"))
-                        .build(),
-                builder().from("log-file")
-                        .to("quarkus.log.file.path")
-                        .defaultValue(DEFAULT_LOG_PATH)
-                        .description("Set the log file path and filename.")
-                        .paramLabel("<path>/<file-name>.log")
-                        .transformer(LoggingPropertyMappers::resolveFileLogLocation)
-                        .build(),
-                builder().from("log-file-format")
-                        .to("quarkus.log.file.format")
-                        .defaultValue("%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n")
-                        .description("Set a format specific to file log entries.")
-                        .paramLabel("<format>")
                         .build()
         };
     }
 
-    private static BiFunction<String, ConfigSourceInterceptorContext, String> resolveLogHandler(String handler) {
+    private static BiFunction<String, ConfigSourceInterceptorContext, String> ResolveLogHandler(String handler) {
```

Comment:
Please, avoid such changes if not strictly related to the scope of the PR.

Reply:
Without this change the code won't compile, if you prefer a different encoding I'm up for changing it, but this (or a similar) change is strictly necessary.